packages/components/src/index.tsx

```diff
@@ -14,3 +14,4 @@ export { Link } from './components/Link/Link'
 export { LoadingIndicator } from './components/LoadingIndicator/LoadingIndicator'
 export { SegmentedControl } from './components/SegmentedControl/SegmentedControl'
 export { Spacer } from './components/Spacer/Spacer'
+export { useTheme } from './utils'
```

Comment:
Nitpicky, but should we split out a separate section (e.g. `// Export consumer available utilities here so they are exported through npm`) for non-component exports? 

Seems like a good idea to keep some order moving forward as the components list grows and potentially other utilities join the exports.